.github/ISSUE_TEMPLATE/icon_update.yml

```diff
@@ -8,11 +8,11 @@ body:
     attributes:
       value: |
         Before opening a new issue, make sure it isn't covered by an existing issue.
-        Please search for [issues with the `icon outdated` label][icon-outdated-issues]
+        Please search for [issues with the `update-icon/data` label][update-icon/data]
```

Comment:
```suggestion
        Please search for [issues with the `update icon/data` label][update-icon/data]
```

Reply:
Yes, let's drop the PR in favor of @adamrusted's #10804.